shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -46,6 +47,12 @@
 @NotThreadSafe
 public class TfsShell implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static final HashMap<String, String[]> CMD_ALIAS = new HashMap<String, String[]>() { {
+      put("chgrpr", new String[] {"chgrp", "-R"});
```

Comment:
also, what do you think about keeping the ChgrpRecursiveCommand class and simply have it extend ChgrpCommand class, overriding run ... I am not sure we need to introduce a new concept of "aliases" just to deprecate an API